opencontext/managers/consumption_manager.py

```
@@ -202,8 +202,10 @@ def check_and_generate_daily_report():
                     try:
                         end_time = int(now.timestamp())
                         start_time = int((now - timedelta(days=1)).timestamp())
-                        
+
                         report_content = asyncio.run(self._activity_generator.generate_report(start_time, end_time))
```

Ruff (0.13.3)

Local variable `report_content` is assigned to but never used

Remove assignment to unused variable `report_content`

(F841)